src/app/code/local/Aligent/GeoIP/Helper/Data.php

```diff
@@ -16,7 +16,7 @@ class Aligent_GeoIP_Helper_Data extends Mage_Core_Helper_Abstract {
     const VARNISH_XGEOIP_SERVER_VARIABLE = 'HTTP_X_GEOIP';
     const VARNISH_XGEOIP_SERVER_HEADER = 'X-GeoIP';
     const CLOUDFLARE_GEOIP_SERVER_HEADER = 'CF-IPCountry';
```

Comment:
@michaelwylde do we need to change server header too ? or it is ok ?

If its ok Happy to merge it.